kitsune/community/templates/community/email/first_answer.ltxt

```diff
@@ -0,0 +1,58 @@
+{# This is an email.  Whitespace matters! #}
+{% autoescape false %}
+{% trans %}
+Thank you for your first contribution to the Mozilla support forums! You just
+made somebody's day!
+{% endtrans %}
+
+{% trans %}
+Mozilla support is fully powered by our volunteer community that is helping
+hundreds of users everyday. This community is formed by passionate Firefox and
+Thunderbird users who take some of their spare time to help others. We couldn't
+do it without your passion and enthusiasm. Your contribution, as minor as it
+might seem, has tremendous value. Solving one issue in the support forums can
+help up to 1000 users a day!
+{% endtrans %}
+
+---
+{{ _('Tell us about yourself!') }}
+
+{% trans %}
+We are always very happy to hear from new people who like helping on the forums
+so If you haven’t met our community yet please take a minute to introduce
+yourself. If you want to stay low-profile, that’s also ok :). You can also find
+us all hanging out on IRC, on the #sumo channel on irc.mozilla.org.
+{% endtrans %}
+
+{% trans url='https://support.mozilla.org/en-US/forums/contributors' %}
+The community forum is at {{ url }}.
+{% endtrans %}
+
+---
+{{ _('Learn new things') }}
+
+{% trans %}
+If you are interested in learning more about how this whole support thing
+works, check out our contributor training. This includes some cool
+troubleshooting info, guidelines, how to questions, and info about where to
+find more help if stuck.
+{% endtrans %}
+https://support.mozilla.org/en-US/kb/introduction-contributor-quality-training
+
+---
+{{ _('Stay in touch') }}
+
+{% trans %}
+Finally, if you want to follow all news related to SUMO, don’t forget about our
+{% endtrans %}
+
+* {{ _('Community forums') }} (https://support.mozilla.org/forums/contributors
+* {{ _('Official blog') }} (http://blog.mozilla.org/sumo/)
+* {{ _('Twitter account') }} (http://twitter.com/sumo_mozilla)
+* {{ _('Weekly meetings') }} (https://wiki.mozilla.org/Support/Weekly_Meetings)
+
+{% trans %}
+Thank you once more. We are looking forward to seeing you more often around
+SUMO! Keep rocking!
```

Comment:
Ditto here about awkward phrasing and how I think this should be "looking forward to seeing you around SUMO more often!"